dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -879,7 +879,6 @@ input.header_input {
 
   .video_name {
     text-align: center;
-    width: 183px;
```

Comment:
Yep! I checked that it still gets the right width when thumbnails are present (because the size of the parent is determined by the thumbnail image).